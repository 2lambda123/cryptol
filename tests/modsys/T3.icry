:module T3::Main




