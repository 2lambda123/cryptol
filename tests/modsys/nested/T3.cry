module T3 where

import submodule A
x = y

submodule A where
  y = x




