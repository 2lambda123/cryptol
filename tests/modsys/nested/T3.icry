:load T3.cry
