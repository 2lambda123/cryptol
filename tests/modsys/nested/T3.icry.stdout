Loading module Cryptol
Loading module Cryptol
Loading module T3

[error] Invalid recursive dependency:
    • T3::x , defined at 4:1--4:2
    • submodule T3::A , defined at 6:11--6:12
